### PR TITLE
Fix path name

### PR DESCRIPTION
**Pull request by ysimonson**
This should fix the netlify docs build errors

### CODE REVIEW

File: doc/build.sh [hunk 1]
```diff
@@ -48,6 +48,8 @@ EOF
 
 # Rebuild all docs versions
 for d in docs/*; do
+    d=$(basename "${d}")
+
     # don't rebuild archive dir
```
Comment (svekars) [inline doc/build.sh]: lines 53 - 60 don't work now 
Screen Shot 2020-03-18 at 7 54 47 AM

It should exclude `docs/` and the `master` and `archive` dirs:

Screen Shot 2020-03-18 at 7 57 39 AM

**Comment by ysimonson** [comment]
Do you know where that menu is generated? Because the build console is indicating that this change fixed at least the aforementioned issue: https://app.netlify.com/sites/pachyderm-docs/deploys/5e722e90518f790009610675

**Comment by ysimonson** [comment]
Ah! I see, there's two loops - going to try another fix.

**Comment by svekars** [comment]
Looking good now!

**Pull request merged**